Pugixml linking error

The build process appears to be broken if pugixml is already installed elsewhere on the system. I'm getting the following error when I try to compile a second version of OpenMC on the same system:

`[ 97%] Linking CXX shared library lib/libopenmc.so
/usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: cannot find -lpugixml
collect2: error: ld returned 1 exit status
make[2]: *** [CMakeFiles/libopenmc.dir/build.make:1430: lib/libopenmc.so] Error 1
make[1]: *** [CMakeFiles/Makefile2:189: CMakeFiles/libopenmc.dir/all] Error 2
make: *** [Makefile:149: all] Error 2`

Perhaps this was caused by #1770?

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -54,13 +54,13 @@ endif()
 #===============================================================================
 
 # If not found, we just pull appropriate versions from github and build them.
-find_package(fmt QUIET)
+find_package(fmt QUIET NO_SYSTEM_ENVIRONMENT_PATH)
 if(fmt_FOUND)
   message(STATUS "Found fmt: ${fmt_DIR} (version ${fmt_VERSION})")
 else()
   message(STATUS "Did not find fmt, will use submodule instead")
 endif()
-find_package(pugixml QUIET)
+find_package(pugixml QUIET NO_SYSTEM_ENVIRONMENT_PATH)
 if(pugixml_FOUND)
   message(STATUS "Found pugixml: ${pugixml_DIR}")
 else()

